packages/calling/calling/src/Calling/Calling.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable tsdoc/syntax */
+import {ISDKConnector, WebexSDK} from '../SDKConnector/types';
+import {CallingConfig, ICalling, WebexConfig} from './types';
+import {createClient} from '../CallingClient/CallingClient';
+import {createContactsClient} from '../Contacts/ContactsClient';
+import {createCallHistoryClient} from '../CallHistory/CallHistory';
+import {createCallSettingsClient} from '../CallSettings/CallSettings';
+import {createVoicemailClient} from '../Voicemail/Voicemail';
+import {ICallingClient} from '../CallingClient/types';
+import {IContacts} from '../Contacts/types';
+import {ICallHistory} from '../CallHistory/types';
+import {ICallSettings} from '../CallSettings/types';
+import {IVoicemail} from '../Voicemail/types';
+import SDKConnector from '../SDKConnector';
+import {Webex} from '../init';
+import {CallingEventTypes, EVENT_KEYS} from '../Events/types';
+import {Eventing} from '../Events/impl';
+import log from '../Logger';
+import {CALLING_FILE} from '../CallingClient/constants';
+
+/**
+ *
+ */
+class Calling extends Eventing<CallingEventTypes> implements ICalling {
+  private callingConfig: CallingConfig;
+
+  private sdkConnector: ISDKConnector;
+
+  private webex: WebexSDK;
+
+  public callingClient!: ICallingClient;
+
+  public contactClient!: IContacts;
+
+  public callHistoryClient!: ICallHistory;
+
+  public callSettingsClient!: ICallSettings;
+
+  public voicemailClient!: IVoicemail;
+
+  /**
+   *
+   * @param callingConfig
+   * @param webex
+   * @param webexConfig
```

Comment:
going forward we should stop adding jsdocs in ts file.

Reply:
Ohh I didn't add this..seems to have added on its own when i ran lint --fix command. Let me fix it